docs/user-guide/grpc.md

```diff
@@ -121,8 +133,24 @@ Next we will modify gRPC Client to use our Træfik self-signed certificate:
 ```go
 // ...
 
+import (
```

Comment:
Could you remove this block (imports) ?

Reply:
we have to add import with your modification

Reply:
please remove